domains/nermh.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Nermhy",
+        "email": "zxc1007csgo@gmail.com"
+    },
+    "record": {
+        "A": ["88.99.214.225"]
```

Comment:
cant reach site (88.99.214.225 refused to connect.)

Reply:
how to connect vercel and this domain?

Reply:
we do not support vercel, please use another hosting service or if you are fine use URL records (tho you wont be able to connect your domain to your vercel website since URL is redirecting)